src/document/DocumentCommandHandlers.js

```diff
@@ -158,8 +158,8 @@ define(function (require, exports, module) {
         var currentDoc          = DocumentManager.getCurrentDocument(),
             windowTitle         = brackets.config.app_title,
             currentlyViewedFile = MainViewManager.getCurrentlyViewedFile(MainViewManager.ACTIVE_PANE),
-            currentlyViewedPath = currentlyViewedFile.fullPath,
```

Comment:
Good fix 👍  Actually one console error was there because of this line on project change. Can you please verify that as well?

Reply:
Yes, this will fix the console error also.